README.md

```diff
@@ -3,8 +3,9 @@
 The purpose of this repository is to store a community-edited, formal description of the architecture of Sigstore. 
 
    * [Sigstore Client Spec](client-spec.md) - This document specifies an architecture for using an automated certificate authority specifically, timestamping service, and transparency service for signing digital payloads.
-   * [Fulcio, A Certificate Authority for Code Signing](fulcion-spec.md) - This document describes Fulcio, a certificate authority for issuing short-lived code signing certificates for an OpenID Connect (OIDC) identity, such as an email address. 
-   * [Rekor, A Transparency Service](rekor-spec.md) - This document describes Rekor, a signature tranparency service that securely records and makes verifiable the metadata of signed software artifacts, ensuring trust and integrity in the software supply chain. 
+   * [Fulcio, A Certificate Authority for Code Signing](fulcio-spec.md) - This document describes Fulcio, a certificate authority for issuing short-lived code signing certificates for an OpenID Connect (OIDC) identity, such as an email address. 
+   * [Rekor, A Transparency Service](rekor-spec.md) - This document describes Rekor, a signature tranparency service that securely records and makes verifiable the metadata of signed software artifacts, ensuring trust and integrity in the software supply chain.
+      * [Rekor 2.0 Proposal](https://docs.google.com/document/d/1Mi9OhzrucIyt-UCLk_FxO2_xSQZW9ow9U3Lv0ZB_PpM/edit?resourcekey=0-4rPbZPyCS7QDj26Hk0UyvA&tab=t.0#heading=h.bjitqo6lwsmn) - ⚠️ Sigstore is moving towards a new design for Rekor 2.0. This change is imminent and a spec doc will be made available in this repo in due course once the community makes the transistion. (To access the proposal doc you must be a member of the [sigstore-dev@ Google group](https://groups.google.com/g/sigstore-dev))    
```

Comment:
Could we include a brief one-line summary of the changes in Rekor 2.0, namely the move towards a tile-based architecture? We can link to the transparency.dev article about the benefits of tiles as well

Reply:
Great suggestions, done!